python/common/org/python/types/Str.java

```diff
@@ -726,28 +726,40 @@ private long toPositiveIndex(long index) {
         __doc__ = ""
     )
     public org.python.Object isalnum() {
-        throw new org.python.exceptions.NotImplementedError("isalnum() has not been implemented.");
+		if (this.value.matches("(\\p{N}|\\p{L})+")) {
```

Comment:
@jack17529 check out the first timers guide: http://pybee.org/contributing/how/first-time/what/voc/